lib/config-initializer.js

```diff
@@ -128,11 +129,87 @@ function promptUser(callback) {
             name: "format",
             message: "What format do you want your config file to be in?",
             default: "JSON",
-            choices: ["JSON", "YAML"]
+            choices: ["JSON", "YAML"],
+            when: function(answers) {
```

Comment:
Right, there are now two separate question flows, so this is at the end of the first one (that was to avoid having to add  `when() { return answers.source !== "guide" }` to every single additional question